### PR TITLE
[1LP][RFR][NOTEST] Add worker appliance to remote region option.

### PR DESCRIPTION
**Pull request by OpCode1300**
Adds option for remote worker in remote region

### CODE REVIEW

File: cfme/scripting/setup_env.py [hunk 1]
```diff
@@ -118,10 +118,13 @@ def setup_ha_env(cfme_version, provider, lease, desc):
...
 @click.option('--sprout-poolid', default=None, help='Specify ID of existing pool')
 @click.option('--desc', default='Replicated appliances', help='Set description of the pool')
-def setup_replication_env(cfme_version, provider, lease, sprout_poolid, desc):
+@click.option('--remote-worker', is_flag=True, help='Add node to remote region')
+def setup_replication_env(cfme_version, provider, lease, sprout_poolid, desc, remote_worker=False):
     lease_time = tot_time(lease)
     """Multi appliance setup with multi region and replication from remote to global"""
     required_app_count = 2
+    if remote_worker:
+        required_app_count = required_app_count + 1
```
Comment (quarckster) [inline cfme/scripting/setup_env.py]: **OPTIONAL**
```python
required_app_count += 1
```

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/scripting/setup_env.py [hunk 2]
```diff
@@ -118,11 +118,16 @@ def setup_ha_env(cfme_version, provider, lease, desc):
 @click.option('--lease', default='3h', help='set pool lease time, example: 1d4h30m')
 @click.option('--sprout-poolid', default=None, help='Specify ID of existing pool')
 @click.option('--desc', default='Replicated appliances', help='Set description of the pool')
-def setup_replication_env(cfme_version, provider, lease, sprout_poolid, desc):
+@click.option('--remote-worker', is_flag=True, help='Add node to remote region')
```
Comment (mshriver) [inline cfme/scripting/setup_env.py]: http://click.pocoo.org/5/options/#boolean-flags

Looking at the documentation of `is_flag` for click, the default is False. I think you should remove the default parameter statement `remote_worker=False` in the function definition below and just add the `remote_worker` option.

File: cfme/scripting/setup_env.py [hunk 3]
```diff
@@ -148,19 +153,30 @@ def setup_replication_env(cfme_version, provider, lease, sprout_poolid, desc):
...
+
     command_set1 = ('ap', '', opt, '2', ip0, '', pwd, '', '1', 'y', '1', 'n', '1', pwd,
         TimedCommand(pwd, 360), '')
     apps[1].appliance_console.run_commands(command_set1)
     apps[1].wait_for_evm_service()
     apps[1].wait_for_web_ui()
     print("Remote region appliance provisioned and configured {}".format(ip1))
+
+    if remote_worker:
+        ip2 = apps[2].hostname
```
Comment (mshriver) [inline cfme/scripting/setup_env.py]: Please add print statement here that remote worker provisioning/configuration starting

File: cfme/scripting/setup_env.py (hunk 2 above)
Comment (mshriver) [reply cfme/scripting/setup_env.py]: Resolved with `d5d70ad`

**Pull request merged**